test/e2e/invalid_test.go

```diff
@@ -78,6 +78,72 @@ func TestInvalidBuild(t *testing.T) {
 			Steps:   []corev1.Container{{Image: "busybox"}},
 			Timeout: &metav1.Duration{time.Duration(36 * time.Hour)},
 		},
+	}, {
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: buildTestNamespace,
+			Name:      "source-with-duplicate-subpath",
+		},
+		Spec: v1alpha1.BuildSpec{
+			Sources: []v1alpha1.SourceSpec{{
+				TargetPath: "paht/foo",
```

Comment:
Is this a typo for path? If not can you choose a name that's more distinct from path?

Reply:
I will fix that.